agent/fw_laravel.c

```diff
@@ -619,7 +619,7 @@ NR_PHP_WRAPPER(nr_laravel5_exception_report) {
       nr_status_t st;
 
       st = nr_php_error_record_exception(NRPRG(txn), exception, priority,
-                                         NULL /* use default prefix */,
+                                         true, NULL /* use default prefix */,
```

Comment:
With the new argument added, this comment is now confusing. Is this comment about 'true' or 'NULL'?